code/src/App.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
+import { Form } from "./js components/Form"
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
+    <>
+      <Form />
+    </>
+  );
 }
```

Comment:
Clean code! I like that you have made components out of each question and therefor are able to just insert the form here.